README.md

```diff
@@ -10,9 +10,9 @@ Rancher is an open source project that provides a container management platform
 
 ## Latest Release
 
-* Latest - v2.5.1 - `rancher/rancher:latest` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.1).
+* Latest - v2.5.2 - `rancher/rancher:latest` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.2).
 
-* Stable - v2.4.5 - `rancher/rancher:stable` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.4.5).
+* Stable - v2.5.2 - `rancher/rancher:stable` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.2).
 
 To get automated notifications of our latest release, you can watch the announcements category in our [forums](http://forums.rancher.com/c/announcements), or subscribe to the RSS feed `https://forums.rancher.com/c/announcements.rss`.
 
```

Comment:
Readme is fine. Is there nothing else that needs to change in master, is RKE/vue up to date ?